all-spark-notebook/Dockerfile

```diff
@@ -6,6 +6,10 @@ FROM $BASE_CONTAINER
 
 LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
+# Fix: https://github.com/hadolint/hadolint/wiki/DL4006
+# Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
It's easier to add this to every Dockerfile.
This way hadolint and anyone who reads the docker file will know we're using bash with pipefail.